lookout/style/format/quality_report_analyzer.py

```diff
@@ -0,0 +1,159 @@
+import logging
+from typing import Any, List, Mapping
+
+import numpy
+
+from lookout.core.analyzer import ReferencePointer
+from lookout.core.api.service_analyzer_pb2 import Comment
+from lookout.core.api.service_data_pb2_grpc import DataStub
+from lookout.core.data_requests import with_changed_uasts_and_contents, with_uasts_and_contents
+from lookout.style.format.analyzer import FormatAnalyzer
+from lookout.style.format.diff import find_new_lines
+from lookout.style.format.feature_extractor import FeatureExtractor
+from lookout.style.format.feature_utils import CLASSES
+from lookout.style.format.model import FormatModel
+from lookout.style.format.quality_report import generate_report
+from lookout.style.format.utils import merge_dicts
+
+MODEL_LOC = "/tmp/home/egor/workspace/tmp/freeCodeCamp/style.format.analyzer.FormatAnalyzer_1.asdf"
+
+
+class QualityReportAnalyzer(FormatAnalyzer):
+    """
+    Quality report analyzer:
+    * analyze - generate report per changed file.
+    * train - generate report per changed file.
+    """
+    log = logging.getLogger("QualityReportAnalyzer")
+    model_type = FormatModel  # TODO - fix if it's required
```

Comment:
Declare a dummy model, or even better, create a DummyAnalyzerModel in `lookout.core`